source/_posts/2017/2017-02-18-symfony-validator-conditional-constraints.md

```diff
@@ -1,7 +1,6 @@
 ---
-layout: post
+id: 2
```

Comment:
- `layout` is now defined in config for posts, `_layouts/post.latte/twig` is the default one
- posts require ids, to better cooperation and interconnection